src/REPLMode.jl

```diff
@@ -763,7 +763,7 @@ end
 
 function complete_local_path(s, i1, i2)
     cmp = REPL.REPLCompletions.complete_path(s, i2)
-    [REPL.REPLCompletions.completion_text(p) for p in cmp[1]], cmp[2], !isempty(cmp[1])
+    filter(isdir, [REPL.REPLCompletions.completion_text(p) for p in cmp[1]]), cmp[2], !isempty(cmp[1])
```

Comment:
Not that it matters but can use `filter!` here. Also, perhaps this could be split into two statements, it is becoming a bit too nested imo (just add a variable name to the list and then `filter!` it).